azure-iot-hub/tests/test_iothub_registry_manager.py

```diff
@@ -0,0 +1,800 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module should be replaced when there are actual tests for the azure.iot.hub library"""
+
+import pytest
+from azure.iot.hub.iothub_registry_manager import IoTHubRegistryManager
+from azure.iot.hub.protocol.iot_hub_gateway_service_ap_is20190630 import (
+    IotHubGatewayServiceAPIs20190630,
+)
+
+from azure.iot.hub.auth import ConnectionStringAuthentication
+
+from azure.iot.hub.protocol.models import AuthenticationMechanism
+
+
+fake_shared_access_key = "Zm9vYmFy"
+fake_shared_access_key_name = "alohomora"
+
+fake_primary_key = "petrificus"
+fake_secondary_key = "totalus"
+fake_primary_thumbprint = "HELFKCPOXAIR9PVNOA3"
+fake_secondary_thumbprint = "RGSHARLU4VYYFENINUF"
+fake_hostname = "beauxbatons.academy-net"
+fake_device_id = "MyPensieve"
+fake_module_id = "Divination"
+fake_managed_by = "Hogwarts"
+fake_etag = "taggedbymisnitryogmagic"
+fake_status = "flying"
+
+
+@pytest.fixture(scope="function")
+def mock_service_operations(mocker):
+    mock_service_operations_init = mocker.patch(
+        "azure.iot.hub.protocol.iot_hub_gateway_service_ap_is20190630.ServiceOperations"
+    )
+    return mock_service_operations_init.return_value
+
+
+@pytest.fixture(scope="function")
```

Comment:
Not wrong, but this is already default behavior, you don't strictly need to specify.